httpchecker.go

```diff
@@ -54,6 +54,23 @@ type HTTPChecker struct {
 	// requests. If not set, DefaultHTTPClient is
 	// used.
 	Client *http.Client `json:"-"`
+
+	// Headers is a slice of Headerthat contains the
+	// name and value for the header.
+	// Default is no additional headers are sent.
```

Comment:
Yeah thats better for sure. Added vs Set is an interesting question, I look at it as added given its coming from the config